russian/call-mom/call-back/call-back.md

```diff
@@ -0,0 +1 @@
+Вы снова набираете номер, однако механический голос сообщает - "4815162342 Данный номер не обслуживается. Авто-система связи станции Лебедь" 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.